deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -338,6 +344,11 @@ func (c *Controller) createOrUpdateReconcile(ctx context.Context, roMR *v1alpha1
 	// Or create a copy manually for better performance
 	mr := roMR.DeepCopy()
 
+	err := c.sendDocumentation(ctx, mr)
```

Comment:
I guess we have to build documentation only for deployed modules. So this function should be removed to the `case v1alpha1.PhaseDeployed:` like that:
```
	case v1alpha1.PhaseDeployed:
		err := c.sendDocumentation(ctx, mr)
		if err != nil {
			return ctrl.Result{Requeue: true}, fmt.Errorf("send documentation error: %v", err)
		}

		// add finalizer and status label
		if !controllerutil.ContainsFinalizer(mr, fsReleaseFinalizer) {
			controllerutil.AddFinalizer(mr, fsReleaseFinalizer)
		}

....
```

Reply:
Also we have to call this method in the override-controller, when we deploy the module somehere [here](https://github.com/deckhouse/deckhouse/blob/main/deckhouse-controller/pkg/controller/module-controllers/release/override-controller.go#L299)